package/src/components/AddressForm/v1/AddressForm.js

```diff
@@ -389,26 +394,15 @@ class AddressForm extends Component {
 
           <ColHalf>
             <Field name="region" label="Region" labelFor={regionInputId} isRequired>
-              {this.regionOptions && this.regionOptions.length > 1 ? (
-                <Select
-                  id={regionInputId}
-                  alphabetize
-                  isSearchable
-                  name="region"
-                  options={this.regionOptions}
-                  placeholder="Region"
-                  isOnDarkBackground={isOnDarkBackground}
-                  isReadOnly={isSaving}
-                />
-              ) : (
-                <TextInput
-                  id={regionInputId}
-                  name="region"
-                  placeholder="Region"
-                  isOnDarkBackground={isOnDarkBackground}
-                  isReadOnly={isSaving}
-                />
-              )}
+              <RegionInput
+                id={regionInputId}
+                options={this.regionOptions}
+                isOnDarkBackground={isOnDarkBackground}
+                isReadOnly={isSaving}
+                name="region"
+                placeholder="Region"
+                value={value ? value.region : null}
```

Comment:
You should not pass `value` here because it will override built-in ReactoForm logic. ReactoForm handles all input values.